iphone/Maps/Core/Routing/MWMRouter.mm

```diff
@@ -370,13 +371,13 @@ + (void)routeAltitudeImageForSize:(CGSize)size completion:(MWMImageHeightBlock)b
     NSData *imageData = router.altitudeImagesData[sizeValue];
     if (!imageData) {
       std::vector<uint8_t> imageRGBAData;
-      int32_t minRouteAltitude = 0;
-      int32_t maxRouteAltitude = 0;
+      int32_t totalAscent = 0;
+      int32_t totalDescent = 0;
       measurement_utils::Units units = measurement_utils::Units::Metric;
```

Comment:
Are feet (imperial system) work on iOS in this case?

Reply:
I used uint32_t instead size_t